ctapipe/reco/energy_regressor.py

```diff
@@ -0,0 +1,383 @@
+import numpy as np
+
+from astropy import units as u
+
+from sklearn.ensemble import RandomForestRegressor
+
+
+def pick_any(my_dict):
```

Comment:
No, d.values() does not create copies of the underlying elements, it just keeps references.

Even list(d.values()) does not copy the elements, it keeps new references to them.

This can be checked by doing `list(d.values())[0] is list(d.values())[0]` which is True.